src/commands/appSettings/AzureWebJobsStoragePromptStep.ts

```diff
@@ -14,14 +14,26 @@ export class AzureWebJobsStoragePromptStep<T extends IAzureWebJobsStorageWizardC
     public async prompt(wizardContext: T): Promise<void> {
         const selectAccount: MessageItem = { title: localize('selectAzureAccount', 'Select storage account') };
         const useEmulator: MessageItem = { title: localize('userEmulator', 'Use local emulator') };
+        const skipForNow: MessageItem = { title: localize('skipForNow', 'Skip for now') };
 
         const message: string = localize('selectAzureWebJobsStorage', 'AzureWebJobsStorage must be set in "{0}" to debug non-HTTP triggers locally.', localSettingsFileName);
-        const result: MessageItem = await ext.ui.showWarningMessage(message, { modal: true }, selectAccount, useEmulator);
+
+        const buttons: MessageItem[] = [selectAccount];
+        if (isWindows) {
+            // Only show on Windows until this is fixed: https://github.com/Microsoft/vscode-azurefunctions/issues/1245
+            buttons.push(useEmulator);
+        }
+        buttons.push(skipForNow);
```

Comment:
Should this only have `skipForNow` if it's not Windows?

Reply:
I'm meh on that button, but figured I might as well add it and see what people do in the telemetry. We can remove it if no one picks it